### PR TITLE
feature/ parent-dash-faq-page

### PR DESCRIPTION
**Pull request by MosharrafMusa**
# Description
This page is build for some most frequently asked questions and answers user might ask. If they need more questions they will have another option to contact us.

Fixes #

- What work was done?
 * Built out a page with some most frequent asked question and answer.
- Why was this work done?
 * So that user get their most common answer immediately.
- What feature / user story is it for?
* As a user I would like a clear UI to access common questions and contact staff.
- Any relevant links or images / screenshots

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [X ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [X ] No
- [ ] Not necessary

## Checklist

- [X ] My code follows the style guidelines of this project
- [X ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [X ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X ] New and existing unit tests pass locally with my changes
- [X ] There are no merge conflicts

### CODE REVIEW

**Comment by MosharrafMusa** [comment]
[As a user I would like a page with common questions answered](https://trello.com/c/d3H2zKd6/46-as-a-user-i-would-like-a-page-with-common-questions-answered)

File: src/components/pages/ParentDashFaq/ParentDashFaqContainer.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+              </Panel>
+            </Collapse>
+          </Layout>
+        </div>
+      </Layout>
+    </>
+  );
+};
+
+export default ParentDashFaqContainer;
```
Comment (Robert-Lark) [inline src/components/pages/ParentDashFaq/ParentDashFaqContainer.js:71]: Looks good - great job.

File: src/components/pages/ParentDashFaq/index.js [hunk 2]
```diff
@@ -0,0 +1 @@
+export { default as ParentDashFaq } from './ParentDashFaqContainer';
```
Comment (Robert-Lark) [inline src/components/pages/ParentDashFaq/index.js:1]: good stuff - keeping it consistent with the rest of the file structure of the application.

File: src/index.js [hunk 3]
```diff
@@ -60,6 +61,7 @@ import Wed from './components/pages/AdminDashboard/DevTools/DayComponents/05_Wed
 import Thurs from './components/pages/AdminDashboard/DevTools/DayComponents/06_Thurs';
 import Fri from './components/pages/AdminDashboard/DevTools/DayComponents/07_Fri';
 import DevModeHeader from './components/pages/AdminDashboard/devModeHeader';
+import { ParentDashboardFaq } from './components/pages/ParentDashFaq';
```
Comment (Robert-Lark) [inline src/index.js:66]: looks good.

File: src/styles/less/index.less [hunk 4]
```diff
@@ -11,6 +11,7 @@
 @import 'LandingPage';
 @import 'MissionControl';
 @import 'ParentDashboard';
+@import 'ParentDashFaq';
 @import 'StoryViewer';
 @import 'ProfileModal';
 @import 'ParentLoader';
```
Comment (Robert-Lark) [inline src/styles/less/index.less]: all looks great good work

**Review by Robert-Lark — APPROVED** [review]
Great work - everything looks exactly as it should in keeping with the style of the rest of the application.

**Review by charlene-johnson — APPROVED** [review]
Everything looks good!

**Pull request merged**